express/routes/players.js

```diff
@@ -3,16 +3,21 @@ const httpStatus = require('http-status');
 const { validate } = require('express-validation');
 
 const playersService = require('../services/players');
-const validations = require('../validations/players')
+const validations = require('../validations/players');
+const validationUtils = require('../../utils/express-validation/validationUtils');
```

Comment:
I suggest importing validate() function so the whole construction will be shorter